### DISCUSSION TITLE
In anticipating the new release ;-)

### DISCUSSION THREAD

**Discussion by WilsonBradley** [curif/AgeOfJoy-2022.1#76] [General]
In anticipating the new release ;-)

**Comment by curif** [comment]
Working on a terrible bug that causes the program to hang without sending any messages.
As a result, v0.2.0 is delayed... :(

**Comment by WilsonBradley** [comment]
Me waiting on next release... 
![image](https://user-images.githubusercontent.com/8173893/214445816-33452cd1-c042-424f-b15a-4e9f6aedf9e7.png)

**Reply by WilsonBradley** [reply]
Heya,

I like the new Neon lights addition... ;-)
Is there a menu for changing controller hand, etc.. in the works?

**Reply by curif** [reply]
I've been fighting the Libretro bug, the memory bug continues but it is more stable now.
After publish the v0.2.0 I have these priorities:

* enhancement #85 (v0.2.1)
* CONTRIBUTING.md 
* user configuration (controllers, sound, NPCs, etc)

I will write a roadmap in the discussion area.
